lib/player.rb

```diff
@@ -0,0 +1,16 @@
+class Player
+  attr_accessor :name, :symbol, :positions
+  
+	
+  def initialize(name, symbol)
+    @name = name
+    @symbol = symbol
+    @positions = [] 
```

Comment:
Where did we actually use `@positions` aside from the `move` method below? Or a better way to put it, what's the purpose of `@positions`?